x/evm/keeper/integration_test.go

```diff
@@ -416,13 +411,11 @@ var _ = Describe("Handling a MsgEthereumTx message", Label("EVM"), Ordered, func
 
 	When("EnableCall param is set to false", Ordered, func() {
 		BeforeAll(func() {
-			// Set params to default values
-			defaultParams := evmtypes.DefaultParams()
-			defaultParams.EnableCall = false
-			err := s.network.UpdateEvmParams(defaultParams)
-			Expect(err).To(BeNil())
+			evmGenesis := evmtypes.DefaultGenesisState()
+			evmGenesis.Params.EnableCall = false
+			s = setupIntegrationTestSuite(evmGenesis)
```

Comment:
Why do we create a new chain here instead of updating the params with the new functions?